### ISSUE TITLE
linkAttributes additional attributes do not render expected markup

### ISSUE THREAD

**Issue by acalvino4** [verbb/navigation#287]
**Description**
Using `node.linkAttributes` with additional attributes does not render expected markup

**Steps to reproduce**
1. Include `{{ node.linkAttributes([{attribute: 'class', value: "text-base text-gray-500 hover:text-gray-900"}]) }}` in twig template as directed by the [documentation](https://verbb.io/craft-plugins/navigation/docs/developers/node#linkAttributes).
2. Observe `class="" href="https://example.com/url/path" 0='{"attribute":"class","value":"text-base text-gray-500 hover:text-gray-900"}'` output into your markup.

**Expected Behavior**
Observe `class="text-base text-gray-500 hover:text-gray-900" href="https://example.com/url/path"` output in my markup.

**Additional info**
- Plugin version: 1.4.24
- Craft version: 3.7.37
- Multi-site:No
- Propagate nodes: No

**Additional context**
Since I know `linkAttributes` already injects a `class` attribute based on cp contents (as evidenced by `class=""` in the output above), I tried this using a different attribute and got the same result. So it seems there is either a bug or the documentation is incorrect.

As a side note, a much nicer syntax for this would be
```twig
{{ node.linkAttributes({
    class: "text-base text-gray-500",
    id: "a-heading",
    ...
}) }}
```
since it avoids the verbosity and confusion of `([{`, and the verbosity of `attribute` and `value` in there. Just a thought.

Thanks for looking into this!

**Comment by engram-design** [comment]
I'm pretty sure the docs are wrong, as it should be exactly as you describe:

```
{{ node.linkAttributes({
    class: "text-base text-gray-500",
    id: "a-heading",
    ...
}) }}
```

That should work - apart from merging attributes defined in the node, which is fixed in [1.4.25](https://github.com/verbb/navigation/releases/tag/1.4.25)

**Comment by acalvino4** [comment]
Thanks, I'll try it out! Thanks for such a fast response!
